apps/admin-portal/src/components/server-configurations/account-recovery.tsx

```diff
@@ -156,57 +126,44 @@ export const AccountRecovery: FunctionComponent<AccountRecoveryProps> = (props:
 		makeAccountRecoveryPatchCall(data, successNotification);
 	};
 
-	const saveAccountRecoveryAdvancedConfigs = (accountRecoveryConfigs) => {
+	const saveAccountRecoveryAdvancedConfigs = (configs) => {
 		const data = {
```

Comment:
IMO the indentation is more than 4 spaces which is the standard used. Could we look into this?